### ISSUE TITLE
Bug - insufficient colour contrast

### ISSUE THREAD

**Issue by tessereth** [AusDTO/gov-au-ui-kit#271]
Running aXe accessibility tests gives the following error:

```
       1) color-contrast: Elements must have sufficient color contrast (critical)
           https://dequeuniversity.com/rules/axe/2.0/color-contrast?application=axeAPI
           The following 1 node violate this rule:

               Selector: body > .hero-med.corporate > .wrapper > p
               HTML: <p style="">Summary.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 3.08 (foreground color: #ffffff, background color: #00a1bb, font size: 15.0pt, font weight: normal)
```

If you could fix the colours, that would be great.

**Comment by TrebBrennan** [comment]
Thanks @tessereth
Looks like we need to adjust the font-size if we want to use that colour.
Currently: 20px (regular) in `.hero-med` and  17px (regular) in `.hero-sml`

Suggesting either
1. 24px in `.hero-med` and 19px Bold in `.hero-sml`
2. or change the colour to a more robust one. Eg. `#007ea8` is the closest I can get at AA

@AlzbetaT any input on this one?

---

<img width="744" alt="screen shot 2016-08-09 at 2 45 19 pm" src="https://cloud.githubusercontent.com/assets/15899197/17504872/12135bda-5e40-11e6-838c-4a669bf94821.png">

**Comment by TrebBrennan** [comment]
Result upon consultation is to go for solution 1 above as it looks almost identical

**Comment by maxious** [comment]
We're probably removing this text anyway... https://github.com/AusDTO/gov-au-beta/issues/372

**Comment by petronbot** [comment]
This contrast issue was fixed in 1.7.3
